src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -64,8 +64,6 @@ public function boot()
         ErrorHandler::register(null, false)->throwAt($this->container->getParameter('debug.error_handler.throw_at'), true);
 
         if ($this->container->hasParameter('kernel.trusted_proxies')) {
-            @trigger_error('The "kernel.trusted_proxies" parameter is deprecated since version 3.3 and will be removed in 4.0. Use the Request::setTrustedProxies() method in your front controller instead.', E_USER_DEPRECATED);
```

Comment:
this change should be reverted as the notice is not about the `framework.trusted_proxies` config option but the corresponding kernel parameter, triggering is still relevant. If you think it should be rediscussed, that should be done on 3.3 where it has been introduced.